docs/html/index.html

```diff
@@ -256,7 +256,7 @@
 <details><summary>Setup P2Pool node</summary>
 <ul>
 <li>Download the latest P2Pool <a href="https://github.com/SChernykh/p2pool/releases/latest" target="_blank">release</a>
-<li>[Optional] Open port <span>37889</span> (P2Pool port) or <span>37888</span> (P2Pool mini port) in your firewall to ensure better connectivity
+<li>[Optional] Open port <span>37889</span> (P2Pool p2p port) in your firewall to ensure better connectivity
```

Comment:
Note this change!
For their respective sidechain type, I removed the mention of other sidechain p2p port.